### PR TITLE
Define configuration unit testing

### PR DESCRIPTION
**Pull request by adericbourg**

### CODE REVIEW

File: cmd/ratool/utils.go [hunk 1]
```diff
@@ -26,8 +26,12 @@ func loadSamplesFile(filename string) ([]*model.Sample, error) {
 	if err != nil {
 		return nil, err
 	}
+	return readSamplesFile(file)
+}
+
+func readSamplesFile(reader io.Reader) ([]*model.Sample, error) {
```
Comment (Thib17) [inline cmd/ratool/utils.go]: /readSamplesFile/readSamples/

File: cmd/ratool/unittest.go [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+		if len(line) > 0 {
+			lineComponents := strings.Split(line, " ")
+			lineWithoutTimestamp := lineComponents[0:2]
+			trimmedLines = append(trimmedLines, strings.Join(lineWithoutTimestamp, " "))
+		}
+	}
+	return strings.Join(trimmedLines, "\n")
+}
+
+func makeOutput(testContext *testConfig, graCfg *config.Config) (string, error) {
```
Comment (Thib17) [inline cmd/ratool/unittest.go]: makeSortedOutput

File: cmd/ratool/unittest.go [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+		if r.Type != diffmatchpatch.DiffEqual {
+			// Generate patch-style diff
+			return diff.LineDiffAsLines(trimmedExpected, trimmedActual)
+		}
+	}
+
+	return nil
+}
+
+func trimTimestamps(s string) string {
```
Comment (Thib17) [inline cmd/ratool/unittest.go]: I think it should not be necessary

File: cmd/ratool/utils.go (hunk 1 above)
Comment (adericbourg) [reply cmd/ratool/utils.go]: Done

File: cmd/ratool/unittest.go (hunk 2 above)
Comment (adericbourg) [reply cmd/ratool/unittest.go]: Done

File: cmd/ratool/unittest.go (hunk 3 above)
Comment (adericbourg) [reply cmd/ratool/unittest.go]: Fixed

**Pull request merged**